source/config.js

```diff
@@ -11,6 +11,10 @@ var defaultConfig = {
 
   // The base URL ungit will be accessible from.
   urlBase: "http://localhost",
+  
+  // The URL root path under which ungit will be accesible.
+  // Either blank or starting with a slash and no trailing slash
```

Comment:
I looked for other config params and there does not seem to be any validation for those.
Is there a good place to validate the config?